RELEASE.md

```diff
@@ -0,0 +1,15 @@
+- Check out the repository and pull the latest updates on MASTER (origin or your fork should work)
+- Update the package version in `package.json`.
+- Update `CHANGELOG.md` - remember to update the links at the bottom of the document.
+- Run `npm install`
+- Commit with message `:package: X.X.X` (replacing with version)
+- Push commit to master
+- Run `npm whoami`, making sure you're logged in properly to NPM. If not, `npm login`.
+- Run `scripts/release.sh`
```

Comment:
nothing wrong with what you have, but `npm run release` is handy too.

Reply:
Good point. Updated. Thanks @jgravois